clients/user-services/hibernate-user-service/src/main/java/com/github/ymikevich/hibernate/user/service/model/User.java

```diff
@@ -0,0 +1,189 @@
+package com.github.ymikevich.hibernate.user.service.model;
+
+import com.github.ymikevich.user.service.common.model.Gender;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "app_user")
```

Comment:
hibernate by default uses the naming strategy, splitting words by upper-case letters, but you can change this default behavior to splitting by an underscore symbol and remove all your explicit definitions from entities.